app/client/components/delivery/records/deliveryRecordsProblemContext.tsx

```diff
@@ -0,0 +1,48 @@
+import { createContext } from "react";
+import {
+  DeliveryRecordApiItem,
+  Subscription
+} from "../../../../shared/productResponse";
+import {
+  ContactPhoneNumbers,
+  DeliveryProblemMap,
+  DeliveryRecordsPostPayload
+} from "./deliveryRecordsApi";
+
+export interface DeliveryRecordsProblemType {
+  category?: string;
+  message?: string;
+}
+
+interface DeliveryRecordsProblemContextInterface {
+  subscription: Subscription;
+  subscriptionCurrency: string;
+  productName: string;
+  apiProductName?: string;
+  problemType?: DeliveryRecordsProblemType;
+  affectedRecords: DeliveryRecordApiItem[];
+  deliveryProblemMap: DeliveryProblemMap;
+  isTestUser: boolean;
+  showProblemCredit?: boolean;
+  repeatDeliveryProblem?: boolean;
```

Comment:
for home delivery, I can't seem to see this being set based on if they've reported a problem already in the last 14 days? 
nor it entering the 'escalation' flow for the same reason, similarly I can't seem to see where it enters the 'escalation' flow when `cancelledAt` is true

Reply:
we have another requirement, that it enters the 'escalation' flow when it's `autoRenew = false`

Reply:
did something go wrong with your rebase/commits that wouldn't include some of the `cancelledAt` stuff we did